plugin/pkg/admission/limitranger/admission.go

```diff
@@ -249,6 +249,42 @@ func mergePodResourceRequirements(pod *api.Pod, defaultRequirements *api.Resourc
 		}
 	}
 
+	for i := range pod.Spec.InitContainers {
+		container := &pod.Spec.InitContainers[i]
```

Comment:
I must be missing something....  the code is the exact same for merging init and non init containers in this method...